code/linear-regression/src/Components/Intro.svelte

```diff
@@ -7,9 +7,9 @@
   <p class="body-text">
     Linear Regression is a simple and powerful model for predicting a numeric
     response from a set of one or more independent variables. This article will
-    focus mostly on how the method is used in machine learning, so we won't
-    cover common use cases like casual inference or experimental design. And
-    although it may seem like linear regression is overlooked in modern machine
+    focus on how the method is used in machine learning, so we won't
```

Comment:
Let's keep this as the original:

focus mostly on how the method is used in machine learning, so we won't cover common use cases like casual inference or experimental design. And although it may seem like linear regression is overlooked in modern machine